app/models/pipeline_run_stage.rb

```diff
@@ -53,8 +53,8 @@ def failed?
     job_status == STATUS_FAILED
   end
 
-  def succeeded? # The whole thing completed successfully
-    db_load_status == 1
+  def succeeded?
+    pipeline_run.output_ready?(load_db_command_func)
```

Comment:
Because eventually we will have more load functions (on the result monitor side) than run stages (job monitor side). So I wanted to write `output_ready?` in such a way that it relates directly to a `load` command.

Main use of `output_ready?` is actually in pipeline_run / result_monitor. The function is only secondarily used here in pipeline_run_stages (job monitor) because we know we can kill the batch job if the last output of the stage is present